src/ACEtk/block/details/BaseDistributionData/src/generateBlocks.hpp

```diff
@@ -0,0 +1,13 @@
+void generateBlocks() {
+
+  std::size_t nr = static_cast< std::size_t >( this->XSS( 1 ) );
```

Comment:
Same comments on `nr` and `ne` below.

Reply:
Changed this in the fix/review-1 branch. Also did this in generateXSS where appropriate.